README.md

```diff
@@ -148,15 +148,15 @@ heroku config:set MY_VAR=the_value
 
 #### Using older version of buildpack
 
-Using the above methods always uses the latest version of the buildpack. We attempt to maintain the buildpack for as many old Elixir and Erlang releases as possible. But sometimes it does get hard since there's a matrix of 3 variables involved here (Erlang version, Elixir version and Heroku stack). If your application cannot be updated for some reason and requires an older version of the buildpack then use the [releases](https://github.com/HashNuke/heroku-buildpack-elixir/releases) page to pick a tag to use. Use the buildpack url with the tag name.
+Using the above methods always uses the latest version of the buildpack. We attempt to maintain the buildpack for as many old Elixir and Erlang releases as possible. But sometimes it does get hard since there's a matrix of 3 variables involved here (Erlang version, Elixir version and Heroku stack). If your application cannot be updated for some reason and requires an older version of the buildpack then use the [commits](https://github.com/HashNuke/heroku-buildpack-elixir/commits/master) page to pick a commit to use. Use the buildpack url with the commit SHA.
```

Comment:
Opinion: I lean towards always using a buildpack url with a commit SHA on production apps. That is to prevent the unpleasant moment when your Heroku builds start to fail because the buildpack you use just got updated with a breaking change. Having buildpacks pinned to a specific version is like having your Hex packages pinned to a specific version in `mix.lock`.

Reply:
Totally agree. Perhaps, we should add something in the README saying that?

Reply:
I will submit a PR.